examples/jupyter/README.md

```diff
@@ -0,0 +1,17 @@
+# Jupyter 
+It is rather simple to deploy Jupyter on MANTL. There are some official Docker images that you can use for this purpose (https://github.com/jupyter/docker-stacks). For this example we use the jupyter/minimal-notebook.
+
+
+We wrapped the Marathon submit REST call in a small script: `deploy.sh`. You can use it to deploy Jupyter on your MANTL cluster.
+
+```bash
+./deploy.sh
+```
```

Comment:
We might want to note that the docker image pulling can take upwards of 15 minutes, it might not be the expected behavior.